client/src/remote/Dialog.js

```diff
@@ -113,12 +113,19 @@ export default class Dialog {
       name
     } = options;
 
+    const buttons = [
+      { id: 'save', label: 'Save' },
+      { id: 'discard', label: 'Don\'t Save' },
+      { id: 'cancel', label: 'Cancel' },
+    ];
+
+    // Re-order buttons for linux
+    if (this.backend.getPlatform() == 'linux') {
```

Comment:
Should be `!= windows` I believe, cf. https://github.com/camunda/camunda-modeler/pull/2895#issuecomment-1091429485.

Reply:
We'll have to keep this as is, because newer mac versions are windows style, save first.